fe/fe-core/src/main/java/com/starrocks/sql/optimizer/rule/transformation/materialization/EquationRewriter.java

```diff
@@ -135,6 +159,14 @@ public void addMapping(ScalarOperator expr, ColumnRefOperator col) {
         if (extendedEntry.second != col) {
             equationMap.put(extendedEntry.first, Pair.create(col, extendedEntry.second));
         }
+
+        if (expr instanceof CallOperator && ((CallOperator) expr).getFnName().equals(FunctionSet.TIME_SLICE)) {
+            // mv:    SELECT time_slice(dt, INTERVAL 5 MINUTE) as t FROM table
+            // query: SELECT time_slice(dt, INTERVAL 5 MINUTE) as t FROM table WHERE dt > '2023-06-01'
+            // if '2023-06-01'=time_slice('2023-06-01', INTERVAL 5 MINUTE), can replace predicate dt => t
+            ScalarOperator first = expr.getChild(0);
+            predicateProbMap.put(first, Pair.create(col, new TimeSliceReplaceChecker(((CallOperator) expr))));
```

Comment:
Great Job! we can add more replace checker later. 

`date_trunc` or other const fold functions.

Reply:
+1